subsys/net/lib/ftp_client/src/ftp_client.c

```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (c) 2020 Nordic Semiconductor ASA
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```

Comment:
Please, do not revert back to the old license.

Reply:
updated